source/index.md

```diff
@@ -763,10 +855,13 @@ Declined   |  40000 |  General input error
 
 ### Status message
 
-A status message is included in every response when you make a new transaction.
+A status message may be included in every response when you create a new
```

Comment:
The use of `may` together with `every` sounds weird here. How about `s/every/the`?